Exercise_A/index.html

```diff
@@ -10,7 +10,7 @@
   </head>
   <body>
     <!-- CODE HERE -->
-
+    <a href="about.html"><img src="Avatar.jpg"/></a>
     <script src="" async defer></script>
   </body>
 </html>
```

Comment:
You did a good job adding an anchor tag and linking it to another HTML page. Well done!